core/registry.js

```diff
@@ -73,28 +73,31 @@ exports.DEFAULT = DEFAULT;
 
 /**
  * A name with the type of the element stored in the generic.
- * @param {string} name The name of the registry type.
- * @constructor
  * @template T
  * @alias Blockly.registry.Type
```

Comment:
For the other classes you've moved the `@alias` down to the constructor as well. Not sure which one makes the JsDoc tool happier but I had been assuming it was on the constructor.

Reply:
Done, for consistency. I'm not sure and slightly expect I'll have to do a cleanup pass when I get to API extractor; so far I've been ignoring the JsDoc tool.